File: Seminar6/python/SRP/Employee.py
from datetime import date

class Employee:
    salary = 0

    def __init__(self, name, dob):
        self.name = name
        self.dob = dob

    def get_emp_info(self):
        return f"name - {self.name} , dob - {self.dob}"

    def takeSalary(self, salary):
        self.salary = salary

File: Seminar6/java/src/SRP/Employee.java
package SRP;
import java.sql.Date;

public class Employee {
    private String name;
    private Date dob;
    private int salary;

    public Employee (String name, Date dob) {
        this.name = name;
        this.dob = dob;
    }

    public String getEmpInfo() {
        return "name - " + name + " , dob - " + dob.toString();
    }

    public void takeSalary(int salary) {
        this.salary = salary;
    }
}
